core/net/src/central/rest_api.rs

```diff
@@ -1,22 +1,24 @@
-use actix_web::{HttpResponse, Responder, Scope};
+use actix_web::{web, HttpResponse, Responder, Scope};
 use ya_client_model::{
     net::{Status, NET_API_V2_NET_PATH},
     NodeId,
 };
 
 use crate::error::Result;
 
-pub fn web_scope() -> Scope {
-    actix_web::web::scope(NET_API_V2_NET_PATH).service(get_info)
+pub fn web_scope(node_id: NodeId) -> Scope {
+    actix_web::web::scope(NET_API_V2_NET_PATH)
+        .app_data(web::Data::new(node_id))
+        .service(get_info)
 }
 
 #[actix_web::get("/status")]
-async fn get_info() -> Result<impl Responder> {
+async fn get_info(node_id: web::Data<NodeId>) -> Result<impl Responder> {
     let status = Status {
-        node_id: NodeId::default(),
+        node_id: (**node_id).clone(),
         listen_ip: None,
         public_ip: None,
-        sessions: 0,
+        sessions: 1,
```

Comment:
You can get default id using gsb call. Check in `core/market/src/identity.rs` how to do this.